pkg/transformer/openshift/openshift.go

```diff
@@ -154,6 +284,36 @@ func (o *OpenShift) Transform(komposeObject kobject.KomposeObject, opt kobject.C
 			objects = append(objects, o.initImageStream(name, service))
 		}
 
+		if service.Build != "" {
+			if !hasBuild {
```

Comment:
``hasBuild`` is a flag to check if there's at least one build in the docker compose file. It's is used to run the block L289-L315 only once per compose file, and print the ``"Buildconfig using %s::%s as source."`` log only once in L331. Does it make sense?